TestCases/parallel_regression.py

```diff
@@ -1428,7 +1428,7 @@ def main():
     sp_pinArray_3d_cht_mf_hf_tp.cfg_dir   = "incomp_navierstokes/streamwise_periodic/chtPinArray_3d"
     sp_pinArray_3d_cht_mf_hf_tp.cfg_file  = "configMaster.cfg"
     sp_pinArray_3d_cht_mf_hf_tp.test_iter = 30
-    sp_pinArray_3d_cht_mf_hf_tp.test_vals = [-13.380025, -7.476945, -7.025285, -0.009675, 99.879812, 4.1920e+02]
+    sp_pinArray_3d_cht_mf_hf_tp.test_vals = [-13.380430, -7.476945, -7.025285, -0.009675, 99.879812, 4.1920e+02]
```

Comment:
also noise? if yes, why would the mentioned commit create such noise? Variable initialization in the initializer list or ctor should to my humble knowledge not change any computation, nor should the const-ing do so.

Please understand that I am particularly careful when it comes to these tests

Reply:
That is a very good way to return the help I've given to your PR's, good job mate

Reply:
I do have an actual interest in why the regression test values here change. And I am unable to explain why that is here. I've failed a ton of reg tests because I unknowingly interfered with that code, so I want to avoid a simple problem here by "forcing" nothing more than a second thought. If your verdict is that this change is safe then I am more than fine with that. 

Non-obvious reg-test value changes require at least a comment if not an explanation. That, in my opinion, is their very purpose.

Reply:
See https://github.com/su2code/SU2/pull/1413#discussion_r818738434 for the noise creator :) 

Btw: 
> [](https://github.com/su2code/SU2/pull/1413/commits/0f4977f4b761df87737ac283518870fb7cbbd466)hmm, [0f4977f](https://github.com/su2code/SU2/pull/1413/commits/0f4977f4b761df87737ac283518870fb7cbbd466) must have introduced sth
my comment from above was wrong. In fact [e9421f5](https://github.com/su2code/SU2/pull/1413/commits/e9421f56886e067f654b6e82c9a35927d96259f5) introduced the change. This is because the ordering of commits is commit date and not the date when it was pushed. So naturally one would think the commits here on github represents a timeline of changes on github itself, but that is not the case. Not that this is a crazy insight, but maybe to be kept in mind when searching a bad commit (one should use git bisect for that anyway)